helix-term/src/health.rs

```diff
@@ -230,6 +230,7 @@ pub fn language(lang_str: String) -> std::io::Result<()> {
     for ts_feat in TsFeature::all() {
         probe_treesitter_feature(&lang_str, *ts_feat)?
     }
+        println!("wiki link : https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers#{}",lang_str);
```

Comment:
It looks like this snuck in but should probably be a separate PR.

Atlassian has a good git guide on how to separate out changes with branches: https://www.atlassian.com/git/tutorials/using-branches

Reply:
By mistake, I edit `master` instead of `develop` ...